app/views/community/videos/index.html.haml

```diff
@@ -1,3 +1,4 @@
+- content_for :canonical_url, Exercism::Routes.community_videos_url
```

Comment:
For nearly all of these, you don't need `Exercism::Routes`. You only need that really if you're outside of the view context. The only exception is `published_solution_url` which is a weird custom method.